demos/CMakeLists.txt

```
@@ -47,3 +47,10 @@ if((${CMAKE_SYSTEM_NAME} MATCHES "^(Linux)$") AND
   add_executable(meltdown meltdown.cc)
   target_link_libraries(meltdown safeside)
 endif()
+
+if((${CMAKE_SYSTEM_NAME} MATCHES "^(Linux)$") AND
+    (${CMAKE_SYSTEM_PROCESSOR} MATCHES "^aarch64$"))
```

```suggestion
    (${CMAKE_SYSTEM_PROCESSOR} MATCHES "^(aarch64)$"))
```

for consistency, and makes it easier as we add/remove things from these "lists"